### PR TITLE
New package: pcaudiolib-1.1

### PR DESCRIPTION
**Pull request by joey-commits**

### CODE REVIEW

**Comment by joey-commits** [comment]
Duplicate of 26384

**Comment by joey-commits** [comment]
Original committer @skmpz asked me to reopen my pull request and closed theirs.

**Comment by joey-commits** [comment]
I would appreciate feedback on whether I formatted the patches correctly. These fix some issues when espeakup is used with espeak-ng.

**Comment by joey-commits** [comment]
Is there any chance that someone may have time to comment on this one? I desire to get this into the repos because then I can add espeak-ng which works better with the Orca and espeakup assistive technologies which I use frequently. Espeak-ng is also recently having a new Klatt synthesis functionality which is easier on some folks ears. If someone clears this, I will immediately resolve the conflicts in common/shlibs so the merge can take place.

File: srcpkgs/pcaudiolib/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+homepage="https://github.com/espeak-ng/pcaudiolib"
+distfiles="https://github.com/espeak-ng/pcaudiolib/archive/${version}.tar.gz"
+checksum=699a5a347b1e12dc5b122e192e19f4db01621826bf41b9ebefb1cbc63ae2180b
+
+pre_configure() {
+	./autogen.sh
+}
+
+pcaudiolib-devel_package() {
+	depends="pcaudiolib>=${version}_${revision}"
```
Comment (Duncaen) [inline srcpkgs/pcaudiolib/template:21]:
```suggestion
	depends="${sourcepkg}>=${version}_${revision}"
```

File: srcpkgs/pcaudiolib/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'pcaudiolib'
+pkgname=pcaudiolib
+version=1.1
+revision=1
+wrksrc=${pkgname}-${version}
```
Comment (Duncaen) [inline srcpkgs/pcaudiolib/template:5]: This is the default value for `wrksrc`.

**Comment by joey-commits** [comment]
Thank you very much for the feedback. I have performed the suggested fixes.

**Comment by ericonr** [comment]
This will have to wait a while to appear on all systems, since chromium is currently being built.

**Pull request merged**